stdlib/ballerina-http/src/main/ballerina/http/http_response.bal

```diff
@@ -262,6 +274,22 @@ public function Response::getMultiparts () returns mime:Entity[] | mime:EntityEr
     }
 }
 
+public function Response::setCacheControl() {
```

Comment:
I thought we don't need this. Because if the user set the values, can't we automatically set the header ?